### DISCUSSION TITLE
Appreciation and Some Observations

### DISCUSSION THREAD

**Discussion by remgr** [GerryDazoo/Slinger#159] [Q&A]
Hi Gerry,

First and foremost, thank you for all your great work, and patience.  I have successfully implemented the server for all the slingboxes I have been put in charge of, from family and friends.  Your prompt responses and explanations helped me get this done.

The observations involve my experiences with four different ISP systems.  Two of the four had routers between the modem and PC's.  They worked flawlessly.  The ISP modem/routers that the other two systems I tried (Verizon Fios & Optimum), did not allow any incoming traffic at all.  It didn't matter how many ports I opened, or any other things I tried.  I ran several port checking applications, all reported "failed".   I have solved this issue by setting up an old spare pc, on a real router, with Google Chrome Remote access, and a folder for each slingbox that is "trapped" by the ISP provider.  Passwords have been shared.

Mostly, I want to thank you.

**Comment by GerryDazoo** [comment]
I've run across this issue. It's been resolved in one of two ways. 
1) Setting up a VPN connection to allow access to the server. 
2) Using SSH to login into the router and setup a reverse tunnel to allow access to the Server and/or the slingbox. This of course depends on the router capabilties and the ISP.
